GitUI/CommandsDialogs/WorktreeDialog/FormManageWorktree.cs

```diff
@@ -46,7 +46,7 @@ private void Initialize()
                 var strings = current.Split(' ');
                 if (strings[0] == "worktree")
                 {
-                    currentWorktree = new WorkTree { Path = strings[1] };
+                    currentWorktree = new WorkTree { Path = current.Substring(9) };
```

Comment:
does this need changing? if so, why?
lines 55 and 66 use `strings[1]`

Reply:
For the other lines than "worktree", the values are separated by a space (that explain the `split()`) but for this line, there is just the abolute path of the worktree, that coud contains spaces.

So we must not split the line and just take the first element after "worktree" but all the rest of the line...